pkg/security/jwt/validator_test.go

```diff
@@ -102,7 +118,11 @@ func TestInvalidToken(t *testing.T) {
 	server := newTestJwks()
 	defer server.Close()
 
-	v := jwt.NewValidator(server.URL+uri, &noTLS)
+	transport := http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
Could turn these 5 lines into a function instead of C&P it everywhere

Reply:
Fixed